modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -682,8 +682,9 @@ public class ApiClient {
 
     Entity<?> entity = serialize(body, formParams, contentType);
 
-    Response response;
+    Response response = null;
```

Comment:
@Moscagus I think it's null by default so it's not necessary to initialize with null